plenum/common/stashing_router.py

```diff
@@ -86,28 +87,35 @@ def pop_all(self) -> Iterable[Tuple]:
         return data
 
 
-class StashingRouter:
+class StashingRouter(Router):
     Handler = Callable[..., Optional[int]]
 
     def __init__(self, limit: int):
+        super().__init__()
         self._limit = limit
         self._logger = getlogger()
-        self._handlers = {}  # type: Dict[Type, StashingRouter.Handler]
         self._queues = {}  # type: Dict[int, StashingQueue]
+        self._subscriptions = Subscription()
```

Comment:
I think we will need to have one `Subscription` for both stashers and buses (not a new instance here), so that we can use it to unsubsrcibe for one service only